tests/framework/CodeInterfaceTests/RAVEN/tests

```diff
@@ -5,6 +5,15 @@
    UnorderedCsv = 'Basic/outer_out.csv'
    max_time = 500
  [../]
+ [./Optimizer]
+   type = 'RavenFramework'
+   input = 'optimizer.xml'
+   [./data]
+     type = OrderedCSV
+     output = 'Optimizer/opt_export_0.csv'
+   [../]
+   max_time = 500
```

Comment:
Does it really take this long? It doesn't look like it needed many optimization steps to find the solution on the boundary.

Reply:
On the regression machines, the slowest I saw was 107 seconds, so I will update that.